src/app/lib/constants/styles.js

```diff
@@ -27,6 +27,11 @@ export const GEL_SPACING_DBL = GEL_SPACING * 2;
 export const GEL_SPACING_HLF = GEL_SPACING / 2;
 export const GEL_SPACING_QUAD = GEL_SPACING * 4;
 
+// REM units
+export const REM = '1rem'; // 16px (resisable by the user in their browser settings)
```

Comment:
It would be good to use `GEL_SPACING` to map to `rem`.
e.g. ```
export const GEL_SPACING_HLF = '0.25rem';
export const GEL_SPACING = '0.5rem';
export const GEL_SPACING_DBL = '1rem';
export const GEL_SPACING_QUAD = '2rem';
```
So that we continue to use the GEL units within our components.